pkgs/development/libraries/kf5gpgmepp/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, lib, fetchgit, cmake, kdeFrameworks, qt5, boost, gpgme }:
```

Comment:
qt5 should then go as well (and kdeFrameworks). Then add qtbase and extra-cmake-modules.